src/components/AddressBadge.tsx

```diff
@@ -34,18 +35,19 @@ type AddressBadgeProps = ComponentPropsWithoutRef<typeof Badge> & {
 
 const AddressBadge = ({ address, showHashWhenNoLabel, className, ...props }: AddressBadgeProps) => {
   const { t } = useTranslation('App')
+  const { isPassphraseUsed } = useGlobalContext()
 
   if (!address) return null
 
   return showHashWhenNoLabel && !address.settings.label ? (
     <Hash className={className}>
-      {address.settings.isMain && '★'}
+      {!isPassphraseUsed && address.settings.isMain && '★'}
```

Comment:
As mentioned main addresses no longer have a star

Reply:
Where was it mentioned, I am sorry if I missed it.

Reply:
https://github.com/alephium/desktop-wallet/pull/364#issuecomment-1254982029

Reply:
Now that the "dot" is moved on the right side, I think it's ok to have the star back!